ansible_base/jwt_consumer/common/util.py

```diff
@@ -36,7 +36,11 @@ def validate_x_trusted_proxy_header(header_value: str, ignore_cache=False) -> bo
         logger.exception("Failed to load public key")
         return False
 
-    timestamp, signature = header_value.split('-')
+    try:
+        timestamp, signature = header_value.split('-')
+    except ValueError:
+        logger.warning("Failed to validate x-trusted-proxy-header, malformed, expected value to contain a - and it did not")
```

Comment:
> and it did not

Or it could have _too many_ -s

```
>>> a, b = 'f-o-o'.split('-')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```

Reply:
```python
>>> "f-o-o".split("-", maxsplit=1)
['f', 'o-o']

```